### PR TITLE
Adding more messages to test.

### PR DESCRIPTION
**Pull request by shomavg**

### CODE REVIEW

File: src/Employer/ESFA.DAS.EmployerFeedbackEmail.Data/EmployerFeedbackRepository.cs [hunk 1]
```diff
@@ -103,9 +103,9 @@ public async Task InsertNewSurveyForFeedback(long feedbackId)
                 });
         }
 
-        public async Task UpsertIntoUsers(IEnumerable<User> users)
+        public async Task UpsertIntoUsers(User user)
         {
-            var userDt = UsersToDatatable(users);
+            var userDt = UsersToDatatable(user);
```
Comment (dev4valtech) [inline src/Employer/ESFA.DAS.EmployerFeedbackEmail.Data/EmployerFeedbackRepository.cs]: If we are changing this to update only one user at a time then we should get rid of datatable and update the stored procedure to take specific required fields as parameter.

Comment (VasanthaKasirajan3008) [reply src/Employer/ESFA.DAS.EmployerFeedbackEmail.Data/EmployerFeedbackRepository.cs]: changes been done. review please.

File: src/Employer/ESFA.DAS.EmployerFeedbackEmail.Database/StoredProcedures/UpsertUsers.sql [hunk 2]
```diff
@@ -1,11 +1,20 @@
 ﻿CREATE PROCEDURE [dbo].[UpsertUsers]
-	@UsersDt UserTemplate READONLY
```
Comment (dev4valtech) [inline src/Employer/ESFA.DAS.EmployerFeedbackEmail.Database/StoredProcedures/UpsertUsers.sql]: The UserTemplate is not being used anywhere else, perhaps delete this database type as well.

Comment (VasanthaKasirajan3008) [reply src/Employer/ESFA.DAS.EmployerFeedbackEmail.Database/StoredProcedures/UpsertUsers.sql]: Its been Removed when i created the PR.

Comment (dev4valtech) [reply src/Employer/ESFA.DAS.EmployerFeedbackEmail.Database/StoredProcedures/UpsertUsers.sql]: I meant physically delete the file that defines this type from the database project.

File: src/Employer/ESFA.DAS.EmployerFeedbackEmail.Database/StoredProcedures/UpsertUsers.sql [hunk 3]
```diff
@@ -1,11 +1,20 @@
...
-	USING @UsersDt AS [Source]
-	ON [Target].UserRef = [Source].UserRef
-	WHEN MATCHED THEN UPDATE 
-	SET [Target].FirstName = [Source].FirstName,
-	[Target].EmailAddress = [Source].EmailAddress
-	WHEN NOT MATCHED THEN INSERT (UserRef, FirstName,EmailAddress)
-	VALUES([Source].UserRef, [Source].FirstName,[Source].EmailAddress);
\ No newline at end of file
+BEGIN
+	IF EXISTS (Select * from dbo.Users WHERE UserRef = @UserRef)
```
Comment (cofaulco) [inline src/Employer/ESFA.DAS.EmployerFeedbackEmail.Database/StoredProcedures/UpsertUsers.sql]: As the likely scenario is mostly going to be updates. Might be worth changing this to 
`UPDATE MyTable SET FieldA=@FieldA WHERE Key=@Key

IF @@ROWCOUNT = 0
   INSERT INTO MyTable (FieldA) VALUES (@FieldA)`

File: src/Employer/ESFA.DAS.EmployerFeedbackEmail.Database/Post Deployment/DropUserTemplateTableType.sql [hunk 4]
```diff
@@ -0,0 +1,2 @@
+﻿IF type_id('[dbo].[UserTemplate]') IS NOT NULL
```
Comment (cofaulco) [inline src/Employer/ESFA.DAS.EmployerFeedbackEmail.Database/Post Deployment/DropUserTemplateTableType.sql]: I am not sure this is necessary, I think the dacpac publish handles removing

**Pull request merged**